vmdb/app/models/ems_kubernetes.rb

```diff
@@ -20,8 +20,16 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
-  def self.raw_connect(hostname, port, _service = nil)
-    kubernetes_connect(hostname, port)
+  def supported_auth_types
+    %w(basic)
+  end
+
+  def supports_authentication?(authtype)
```

Comment:
I generally go for consistency first, so if all the others are authtype, then go that way.